pydoctor/test/test_astbuilder.py

```diff
@@ -2015,3 +2015,35 @@ def test_package_name_clash(systemcls: Type[model.System]) -> None:
     builder.addModuleString('', 'sub2', parent_name='mod', is_package=True)
 
     assert isinstance(system.allobjects['mod.sub2'], model.Module)
+
+@systemcls_param
+def test_reexport_wildcard(systemcls: Type[model.System]) -> None:
+    """
+    """
```

Comment:
I tried the explain the scope and purpose of this test :)

```suggestion
def test_reexport_wildcard(systemcls: Type[model.System]) -> None:
    """
    If a target module,
    explicitly re-export via C{__all__} a set of names
    that were initially imported from a sub-module via a wildcard,
    those names are documented as part of the target module.
    """
```